signer/src/transaction_coordinator.rs

```diff
@@ -1028,8 +1040,29 @@ mod tests {
     #[tokio::test]
     async fn should_be_able_to_coordinate_signing_rounds() {
         test_environment()
-            .assert_should_be_able_to_coordinate_signing_rounds()
+            .assert_should_be_able_to_coordinate_signing_rounds(tokio::time::Duration::ZERO)
+            .await;
+    }
+
+    #[tokio::test]
+    async fn should_wait_before_processing_bitcoin_blocks() {
+        // NOTE: Above test `should_be_able_to_coordinate_signing_rounds`
+        // could be removed as redundant now.
```

Comment:
I'm curious why you added this comment instead of just deleting `should_be_able_to_coordinate_signing_rounds`. Are you asking for confirmation that you can delete it?

Reply:
Yep, gathering feedback but also, if in doubt, with tests more is better :)

Reply:
Makes sense, resolved.